packages/lttng-ust/package.py

```diff
@@ -14,12 +14,15 @@ class LttngUst(AutotoolsPackage):
 
     maintainers = ['Kerilk']
 
-    version('2.13.2', sha256='08679a1dfc2c9428885273861628aa5d828972f389a38bb08fdce39adee589b7')
```

Comment:
should we keep 2.13.2?

Reply:
No I don't know how to apply the patch there so for now we should get rid of it.

Reply:
we could add from 2.13.4 on